### PR TITLE
Feature/bypass role

### PR DESCRIPTION
**Pull request by louismunro**
## Description

Allows nodes to have a bypass_role (same as bypass_vlan) that allows unconditional role override on a node by node basis. 
## Impacts

Should return the bypass_role when node is registered regardless of authentication rules.
## NEWS file entries

New Features
++++++++++++
Bypass Role: allows bypassing all authorization rules for a registered node. 
## Delete branch after merge

YES

### CODE REVIEW

**Comment by julsemaan** [comment]
I would maybe rephrased the news file entry as : 
- Added a bypass role to the nodes that bypasses all authorization rules for a registered node.

**Comment by julsemaan** [comment]
That PR also needs rebasing against devel

**Comment by louismunro** [comment]
It's not really finished anyway. 
It needs a few changes in the admin.

**Comment by jrouzierinverse** [comment]
Closing
